app/src/main/java/com/android/streetworkapp/ui/event/AddEvent.kt

```diff
@@ -115,22 +119,31 @@ fun AddEventScreen(
           }
       FloatingActionButton(
           onClick = {
-            if (event.title.isEmpty()) {
+            if (event.date.toDate() < Calendar.getInstance().time) {
```

Comment:
To avoid accepting events in the past, we could authorize events only from the **present moment + 5 minutes**. Because I think that creating an event that takes place at the precise moment of its creation doesn't make sense, and so shouldn't be allowed. What do you think about this?

Reply:
Since the toDate() fonction makes the timeStamp of event.date loose in precision, I am not sure that this addition of 5 minutes would be really useful. For the moment this fix works for obvious mistakes of the user (for instance if he chooses a day before today or a really old date). It is also a fix for the "by default date" bug that we encountered in the last meeting: the date was by default 1970. I think that not allowing an event a the present moment makes sens and we should talk about it in the next meeting. For the scope of this PR, I think the current solution is the most relevant.

Reply:
Thank you for clarifying the problem! Then we'll talk about it again at a future meeting, in which we'll also clarify the default date.